entries/toggle.xml

```diff
@@ -24,7 +24,7 @@
   </signature>
   <signature>
     <added>1.3</added>
-    <argument name="showOrHide" type="Boolean">
+    <argument name="doShow" type="Boolean">
```

Comment:
Not sure that `doShow` does this any better...  How about `setShown` ?  

Also maybe add a description in the `<desc>` tag specifying `true` will show and `false` will hide?